README.md

```diff
@@ -237,7 +237,7 @@ Connects to the broker specified by the given url and options and
 returns a [Client](#client).
 
 The URL can be on the following protocols: 'mqtt', 'mqtts', 'tcp',
-'tls', 'ws', 'wss'. The URL can also be an object as returned by
+'tls', 'ws', 'wss', 'file' (e.g. unix socket). The URL can also be an object as returned by
```

Comment:
`unix` should be used not `file` as it needs to be a unix socket not just a normal file.